presto-main/src/main/java/com/facebook/presto/sql/planner/Partitioning.java

```diff
@@ -123,6 +123,44 @@ public boolean isCompatibleWith(
         return true;
     }
 
+    public boolean isRefinedPartitioningOver(
+            Partitioning right,
+            Metadata metadata,
+            Session session)
+    {
+        if (!handle.equals(right.handle) && !metadata.isRefinedPartitioningOver(session, handle, right.handle)) {
```

Comment:
`handle.equals(right.handle)` - do you think we should leave a comment (or give a better name to the method) to reflect that by `Refined` we mean `SameOrRefined`?  I'm not sure if it is worth the verbosity though.

Reply:
I originally start with `isRefinedPartitioningOverOrEqual`,  but then personally feel it's too wordy...

I think we can have comments state that "RefinedOver` is reflective.  
 
Thoughts? @rschlussel